### ISSUE TITLE
[New Request] Generate .conf outside .zip with the same name of .zip generated

### ISSUE THREAD

**Issue by toniosj** [Voljega/ExoDOSConverter#140]
Example:

.\Lost Eden (1995).zip
.\Lost Eden (1995).conf

Why? It would be interesting if we could use original exodos .zip files, with the .conf files generated by eXoDosConverter.

**Comment by v4u6h4n** [comment]
Agreed. Being able to keep the original zip files without needing a duplicate would really simplify things.
